### DISCUSSION TITLE
Federating servers

### DISCUSSION THREAD

**Discussion by Martinligabue** [rustdesk/rustdesk#6341] [General]
I think it would be nice to have servers talking to each other, so you don't have to change settings to connect to clients on that other network.
I thought it would be opt-in, and the ids would add the server's address to the end, like 12345@google.com or 12345@1.1.1.1.
I haven't found anything similar online, so I'm asking here

**Comment by DMSnz** [comment]
does this cover what you are asking?
already in the nightly build it would seem...
https://github.com/rustdesk/rustdesk/issues/6198

**Reply by Martinligabue** [reply]
Yes, that's it, I didn't find it when I searched for similar issues
